src/community/__stories__/blocks-story.tsx

```diff
@@ -170,8 +176,10 @@ BlockHighlightedValue.args = {
     text: 'text',
     value: '20',
     secondaryValue: '20',
+    Pretitle: 'Pretitle',
     title: 'title',
     description: 'description',
+    strikedValue: 'striked Value',
```

Comment:
just a detail, but I think the order of this list should be:

- headline
- headlineType
- pretitle
- strikedValue
- value
- text
- secondaryValue
- title
- description

to follow de visual order